apollo-router/src/compute_job/mod.rs

```diff
@@ -158,6 +158,11 @@ pub(crate) fn queue() -> &'static AgeingPriorityQueue<Job> {
                 }
             });
         }
+        tracing::info!(
+            threads = pool_size,
+            queue_capacity = QUEUE_SOFT_CAPACITY_PER_THREAD * pool_size,
+            "ComputeJob thread pool created",
```

Comment:
```suggestion
            "compute job thread pool created",
```
there might be an even better way to format this message, but let's atleast move away from PascalCase 😁

Reply:
Good call! Sorry I missed this, must have messed up my notifications settings.